src/injectscripts/messages/create.js

```diff
@@ -125,72 +131,160 @@ $(document).ready(function() {
+                                                    BuildNew = {};
+                                                    BuildNew.Contact = v;
+                                                    BuildNew.ContactTypeId = v.ContactTypeId
+                                                    if (vv.TeamLeader) {
+                                                        BuildNew.Description = v.FirstName + " " + v.LastName + " (TL)";
+
+                                                    } else {
+                                                        BuildNew.Description = v.FirstName + " " + v.LastName;
+
+                                                    }
+                                                    BuildNew.Recipient = v.Detail;
+                                                    msgsystem.selectedRecipients.push(BuildNew)
+                                                }
+                                            })
+                                        }
+                                    }
+                                    button.removeChild(spinner);
+                                    //cb for when they are loaded
+                                    var nodes = button.childNodes;
+                                    for (i = 0; i < nodes.length; i++) {
+                                        nodes[i].removeAttribute("style");
+                                    }
+                                }
+                            })
+                        })
+                    })
+
+                    $(button).appendTo('#lighthouseteams');
+                    button.style.width = button.offsetWidth + "px";
+                    button.style.height = button.offsetHeight + "px";
+                }
+            })
+        } else {
+            //nothing found
+            $('#lighthouseteams').empty() //empty to prevent dupes
+
+        }
+    });
+}
+
+
 function LoadAllCollections() {
+    collectionscount
 
     $("#lighthousecollections").empty();
 
-        //Load the saved Collections
-        theLoadedCollection = JSON.parse(localStorage.getItem("lighthouseContactCollections"));
-        if (theLoadedCollection) {
-            $("#collectionscount").textContent = theLoadedCollection.length;
-            theLoadedCollection.forEach(function(item) {
-                var $button = make_collection_button(item.name,item.description,item.items.length+"")
-                var $spinner = (<i style="margin-top:4px" class="fa fa-refresh fa-spin fa-2x fa-fw"></i>)
-
-                $($button).click(function() {
+    //Load the saved Collections
+    theLoadedCollection = JSON.parse(localStorage.getItem("lighthouseContactCollections"));
+    if (theLoadedCollection) {
+        $("#collectionscount").text(theLoadedCollection.length);
+        theLoadedCollection.forEach(function(item) {
+            var $button = make_collection_button(item.name, item.description, item.items.length + "")
```

Comment:
check the source for `make_collection_button` - if it is not a jQuery object don't prefix it with `$`.